.pre-commit-config.yaml

```diff
@@ -26,17 +25,6 @@ repos:
     hooks:
       - id: reorder-python-imports
 
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v0.761"
-    hooks:
-      - id: mypy
```

Comment:
I thought we believed in types 😢 Would you be ok if I added in the types to keep this check

Reply:
mypy doesn't handle the weird globals and eval nonsense